bellboy/actors/lead.py

```diff
@@ -107,22 +129,29 @@ def receiveMsg_SensorEventMsg(self, message, sender):
                 message.eventData,
             )
         )
-        message = LcdMsg(
+
+        # Form a message based on the SensorEventMsg
+        sensor_message_str = f"Requested Floor #{str(message.eventData)[6]}"
+
+        # Show the message on the LCD
+        lcd_message = LcdMsg(
             LcdReq.DISPLAY,
-            displayText=f"Requested Floor #{str(message.eventData)[6]}",
+            displayText=sensor_message_str,
```

Comment:
is there a way for the message to be displayed until a floor is chosen?
just cause ik sometimes it gets cut off and returns to "welcome to bellboy"

Reply:
That needs to be programmed into the LCD actor. Sure can.

Reply:
@seinizumita add an issue to the repository and we'll get to in in the future.